litestar/app.py

```diff
@@ -478,9 +478,7 @@ def serialization_plugins(self) -> list[SerializationPluginProtocol]:
         return list(self.plugins.serialization)
 
     @staticmethod
-    def _get_default_plugins(plugins: list[PluginProtocol] | None = None) -> list[PluginProtocol]:
-        if plugins is None:
-            plugins = []
```

Comment:
This is only called in one place, and we pass `plugins or []` to it there, so the `None` type handling in here is not necessary.